doc/tech/policies.md

```diff
@@ -0,0 +1,61 @@
+
+CNTT realizes there are implications to these types of prohibitions and practical limitations on the ability to live by the Abstraction Principle. A textbook example of a capability that transgresses this principle, is SR-IOV. Other, less notable, yet very important examples include vGPUs and other acceleration hardware, such as FPGA. Less obvious, yet critically important examples are the programs comprising the actual VNFs. As workloads execute native microarchitecture opcodes, their instruction sets effectively constitute an ABI (Application Binary Interface), which is by definition dependent on the host's hardware. Additionally, these programs may or may not, attempt to execute vendor-specific extensions to standard instruction sets, such as x86 or ARM.
+
+Solving the problems associated with implementing the Abstraction Principle is a work in progress. CNTT has not solved all of the associated problems, nor has the industry. As technology evolves and more designs implement Cloud Native concepts, these problems will be addressed. This document is specifically intended as a one-stop-shop to capture CNTT policies which manage these situations as they exist today and their associated exceptions and transitions, and help drive the technology around and supporting the maturation of the Abstraction Principle.
+
+Several specific technology areas have been identified by CNTT as using an ABI impacted by the Abstraction Principle, as follows:
+- SR-IOV
+- GPU/NPU
+- FPGA/Other Acceleration
+- CPU Instruction Sets and Extensions
+
+The preceding list is not exhaustive; technologies will be added as required.
+
+<a name="7.1.3"></a>
+### 7.1.3 Current CNTT Policies
+
+The following sets of compromises and transition plans comprise the policy for each technology subject to this document.
+
+Be aware the compromises and transition plans contained herein, are directly related to factors which are subject to change with the evolution of technology, with changes in industry direction, with changes in standards, etc. Hence, **the policies are subject to change without notice**, and the reader is advised to consult the latest online Github revision of this chapter. All locally stored, printed or other copies should be considered obsolete.
+
+> **Note to Authors:** Status should be set to "Proposed" when initial content is entered. Once alignment is attained following vetting and discussion, status should be set to "Aligned". Immediately prior to merge, status should be set to "In Force". When amending previously approved language, status should be changed from "In Force" to "In Force (Pending Changes)", followed by "Aligned" and ultimately, "In Force".
+
+**SR-IOV**
```

Comment:
Can we have this as a linkable subchapter?

Reply:
yes, done in: https://github.com/cntt-n/CNTT/pull/1267/commits/74e1f812de489221fecf3d271c7a7254d64b6f28